accounts/usbwallet/hub.go

```diff
@@ -156,11 +162,13 @@ func (hub *Hub) refreshWallets() {
 	}
 	infos, err := usb.Enumerate(hub.vendorID, 0)
 	if err != nil {
+		failcount := atomic.AddUint64(&(hub.enumerationFailures), 1)
 		if runtime.GOOS == "linux" {
 			// See rationale before the enumeration why this is needed and only on Linux.
 			hub.commsLock.Unlock()
 		}
-		log.Error("error enumerating USB enumeration: ", "code", err)
+		log.Error("Error enumerating USB", "code", err, "failcount", failcount,
```

Comment:
Nits:

 * Please change the message to `Failed to enumerate USB devices`, we'll have an `err` field either way.
 * Please change the `code` to `err`, that's teh canonical way in the codebase.
 * Please move `err` to the end (the last field in a log can be arbitrarilly long, if you place the error as the first one, you'll end up with weird alignments).
 * Please move the hub and vendor to be the first and second fields respectively.